src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -240,7 +240,7 @@ public bool TryPersist(OptionKey optionKey, object value)
                 // We store these as strings, so serialize
                 var valueToSerialize = value as NamingStylePreferences;
 
-                if (value != null)
+                if (valueToSerialize != null)
```

Comment:
instead of this:
```C#
var valueToSerialize = value as NamingStylePreferences;	  
if (valueToSerialize != null)
{
    value = valueToSerialize.ToXElement().ToString();
}
```

could we do this?:
```C#
if (value is NamingStylePreferences valueToSerialize )
{
    value = valueToSerialize.ToXElement().ToString();
}
```